### DISCUSSION TITLE
24 ottobre 2022 - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#25] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/8ccf1bac-0153-411c-94b4-b60700da2437)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/97389762/bc7b8965-4b46-4a4a-bb4f-c50ddeb6d3a3)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-14 13:17:00](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/157cc661-39e3-41e3-84ae-4d871004b390)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/6ecd6052-e731-47ed-8652-06a593a32b9e)
